content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -124,10 +124,80 @@ You can also create a Markdown hyperlink by highlighting the text and using the
 
 {% data reusables.repositories.section-links %}
 
+If you need to determine the anchor for a heading in a file you are editing, you can use the following basic rules:
+
+* Letters are converted to lower-case.
+* Spaces are replaced by hyphens (`-`). Any other whitespace characters are removed.
+* Leading and trailing whitespace are removed.
+* Paired formatting markup characters are removed, leaving only their contents (for example, `_italics_` becomes `italics`).
+* If the automatically generated anchor for a heading is identical to an earlier anchor in the same document, a unique identifier is generated by appending a hyphen and an auto-incrementing integer.
+
+For more detailed information on the requirements of URI fragments, see [RFC 3986: Uniform Resource Identifier (URI): Generic Syntax, Section 3.5](https://www.rfc-editor.org/rfc/rfc3986#section-3.5).
+
+The code block below demonstrates the basic rules used to generate anchors from headings in rendered content.
+
+```markdown
+# Example headings 
+
+## Sample Section
+
+## This'll  be a _Helpful_ Section About the Greek Letter Θ!
+A heading containing invalid characters, UTF-8 characters, two consecutive spaces between the first and second words, and formatting.
+
+## This heading is not unique in the file
+
+TEXT 1
+
+## This heading is not unique in the file
+
+TEXT 2
+
+# Links to the example headings above
+
+Link to the sample section: [Link Text](#sample-section).
+
+Link to the helpful section: [Link Text](#thisll--be-a-helpful-section-about-the-greek-letter-Θ).
+
+Link to the first non-unique section: [Name](#this-heading-is-not-unique-in-the-file).
+
+Link to the second non-unique section: [Name](#this-heading-is-not-unique-in-the-file-1).
+```
+
+> [!NOTE]
+> If you edit a heading, or if you change the order of headings with "identical" anchors, you will also need to update any links to those headings as the anchors will change.
+
 ## Relative links
 
 {% data reusables.repositories.relative-links %}
 
+## Custom anchors
+
+You can use standard HTML anchor tags (`<a name="unique-anchor-name"></a>`) to create navigation anchor points for any location in the document.
```

Comment:
```suggestion
You can use standard HTML anchor tags (`<a name="unique-anchor-name"></a>`) to create navigation anchor points for any location in the document. To avoid ambiguous references, use a unique naming scheme for anchor tags, such as adding a prefix to the `name` attribute value.
```